dev-util/crash/crash-7.2.8-r1.ebuild

```diff
@@ -6,7 +6,7 @@ EAPI="7"
 inherit toolchain-funcs
 
 DESCRIPTION="Red Hat crash utility; used for analyzing kernel core dumps"
-HOMEPAGE="https://people.redhat.com/anderson/"
+HOMEPAGE="https://crash-utility.github.io/"
 SRC_URI="https://people.redhat.com/anderson/${P}.tar.gz"
```

Comment:
Since it seems to be hosted now on Github too, maybe change SRC_URI?

Reply:
This is problematic as the GitHub release does not include the bundled `gdb`, so the manifest will be different for the same version, likely leading to issues when mirroring. I have added a commit showing the difference.